packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -679,7 +679,7 @@ void main() {
       await tester.pump(kLongPressTimeout);
 
       // Drag enough to move down the first item
-      await drag.moveBy(const Offset(0, 150));
+      await drag.moveBy(const Offset(0, 50));
```

Comment:
This is to match the value at the line `635`
https://github.com/flutter/flutter/blob/6c1455a23ae53ec3a92be8f50a798d6c789b471f/packages/flutter/test/widgets/reorderable_list_test.dart#L635

It was first `150` in both tests because of a bug discussed in https://github.com/flutter/flutter/pull/81396 and solved in https://github.com/flutter/flutter/pull/82296. However, this value was not updated during https://github.com/flutter/flutter/pull/82296 to match its related test.